src/mpid/ch4/netmod/ofi/ofi_events.c

```diff
@@ -488,12 +488,12 @@ static int am_isend_event(struct fi_cq_tagged_entry *wc, MPIR_Request * sreq)
 
     MPIR_FUNC_ENTER;
 
-    msg_hdr = &MPIDI_OFI_AMREQUEST_HDR(sreq, msg_hdr);
+    msg_hdr = &MPIDI_OFI_AM_SREQ_HDR(sreq, msg_hdr);
```

Comment:
NOTE: this ensures that we access the correct `msg_hdr` from the `sreq` since now the rdma part uses `MPIDI_OFI_AM_RREQ_HDR`, which is separate from the SREQ region.

Reply:
OK, I think I get it now.